app/views/devise/omniauth_callbacks/new.html.haml

```diff
@@ -3,18 +3,19 @@
   = devise_error_messages!
   .col-md-6
     = form_for(resource, as: resource_name, class: 'input-group', url: registration_path(resource_name)) do |f|
-      = hidden_field(:user, :first_name)
-      = hidden_field(:user, :last_name)
-      = hidden_field(:user, :email)
-      = f.hidden_field(:project_id, value: 1)
+      = f.label :first_name
+      = f.text_field :first_name, autofocus: true, class: 'form-control', required: true
+
+      = f.label :last_name
+      = f.text_field :last_name, autofocus: true, class: 'form-control', required: true
 
       = f.label :phone
       = f.telephone_field :phone, class: 'form-control phone-input', autofocus: true, placeholder: '+38(xxx)-xxx-xx-xx', autocomplete: "phone", required: true
 
       .field
-        = f.label :project_id
-        = f.select :project_id, @projects.collect { |p| [ p.name, p.id ] }, {} , class: 'form-control'
-      
+        = f.label :email
+        = f.email_field :email, class: 'form-control', autofocus: true, autocomplete: "email"
```

Comment:
Why do you set `autofocus: true`?

Reply:
--> https://github.com/CyberCraftInc/cyber-dashboard/pull/32/commits/f5ff069ff0c6834ecbfddd653f0c75ceaf18996e